sonar-db/src/main/java/org/sonar/db/user/AuthorDao.java

```diff
@@ -53,12 +53,7 @@ public void insertAuthor(DbSession session, String login, long personId) {
   }
 
   public List<String> selectScmAccountsByDeveloperUuids(final DbSession session, Collection<String> developerUuids) {
-    return DatabaseUtils.executeLargeInputs(developerUuids, new Function<List<String>, List<String>>() {
-      @Override
-      public List<String> apply(List<String> partition) {
-        return getMapper(session).selectScmAccountsByDeveloperUuids(partition);
-      }
-    });
+    return executeLargeInputs(developerUuids, partition -> getMapper(session).selectScmAccountsByDeveloperUuids(partition));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)